src/org/mozilla/javascript/NativeMath.java

```diff
@@ -153,7 +153,7 @@ private static Object atanh(Context cx, Scriptable scope, Scriptable thisObj, Ob
                 }
                 return ScriptRuntime.negativeZeroObj;
             }
-            return Double.valueOf(0.5 * Math.log((x + 1.0) / (x - 1.0)));
+            return Double.valueOf(0.5 * Math.log((x + 1.0) / (1.0 - x)));
```

Comment:
maybe pedantic but to be in sync with MDN

    return Double.valueOf(0.5 * Math.log((1.0 + x) / (1.0 - x)));

Reply:
Done